swap_meet/clothing.py

```diff
@@ -1,2 +1,12 @@
-class Clothing:
-    pass
\ No newline at end of file
+from swap_meet.item import Item
+
+class Clothing(Item):
+    def __init__(self, category = "", condition = 0):
+        super().__init__(category = "Clothing", condition = condition)
+
+    def __str__(self):
+        string_item = "The finest clothing you could wear."
+        return string_item
+
+    def condition_description(self):
```

Comment:
Since Clothing inherits from Item, an instance of clothing will automatically have access to the parent class methods (like condition_description). 

So if you did `pants = Clothing(condition=5)` then you could call `pants.condition_description() even if you had not written lines 11 and 12. 

What's happening on line 11 and 12 is that you create a condition_description method in Clothing which overrides the method h